src/sherpa_ai/actions/arxiv_search.py

```diff
@@ -19,9 +19,9 @@
 class ArxivSearch(BaseRetrievalAction):
     role_description: str
     task: str
-    llm: Any  # The BaseLanguageModel from LangChain is not compatible with Pydantic 2 yet
+    llm: Any = None  # The BaseLanguageModel from LangChain is not compatible with Pydantic 2 yet
```

Comment:
Since we are adapting it to Pydantic 2, should we change the type annotation for the llm to BaseLanguageModel? I'm not sure if now LangChain is compatible with Pydantic 2 now.